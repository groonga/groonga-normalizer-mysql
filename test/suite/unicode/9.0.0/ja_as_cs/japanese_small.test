register normalizers/mysql

normalize \
  'NormalizerMySQLUnicode("version", "9.0.0", "locale", "ja", \
                       "accent_sensitive", true, "case_sensitive", true)' \
  "ã‚‡"
