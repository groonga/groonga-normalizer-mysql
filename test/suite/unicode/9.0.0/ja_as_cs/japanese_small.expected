register normalizers/mysql
[[0,0.0,0.0],true]
normalize   'NormalizerMySQLUnicode("version", "9.0.0", "locale", "ja",                        "accent_sensitive", true, "case_sensitive", true)'   "ょ"
[[0,0.0,0.0],{"normalized":"ょ","types":[],"checks":[]}]
